hedgehog/hedgehog.cabal

```diff
@@ -78,6 +78,7 @@ library
 
   ghc-options:
     -Wall
+    -Werror
```

Comment:
`-Werror` is discouraged in public libraries, since warnings could be introduced by newer versions of GHC, making things a bit too brittle for downstream (I believe `cabal lint` complains about it)